### ISSUE TITLE
[SF][A11y][0.26.1][DynamicPageTitle] Screen reader issues of actions overflow menu button 

### ISSUE THREAD

**Issue by ksblklu** [SAP/ui5-webcomponents-react#3234]
**Describe the bug**
When user opens the actions overflow ellipsis button menu of DynamicPageTitle
1) observe when opened, there is no indication of opening
2) as we navigate through the options, the item count is also missing

**To Reproduce**
Environment: Windows, Chrome, JAWS 2022
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-dynamicpage--default-story
2. Click on actions overflow button of DynamicPageTitle
3. Tab navigating through the popup menu items
4. Inspect the JAWS speech history, theres no indication of overflow popup menu, and item count of menu items is also missing
<img width="1358" alt="image" src="https://user-images.githubusercontent.com/27538066/178435316-65516891-5f78-41c5-9127-c97d0a827cc1.png">

**Expected behavior**
Refer to WCAG actions menu button:
![image](https://user-images.githubusercontent.com/27538066/178437663-3d2d30d2-4cde-4e13-9d17-4d907980ff3e.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 0.26.1
`@ui5/webcomponents-react` version: 1.5.0
Operating System: Windows
Browser: Chrome

**Comment by jatinthaker** [comment]
What is the plan on fixing this issue? Priority of medium is ok. In agreement with Stoehr.

**Comment by Lukas742** [comment]
Hi @ksblklu 

I compared our implementation with the one of SAPUI5 and our announcements are not 100% equal, but similar enough in my opinion. E.g. with SAPUI5 there is also no item count and if the popover is open there is an "dialog" announcement with NVDA, but none with JAWS (which seems faulty to me, since the role is set to `dialog`).

Since we are following the UI5 implementation, I think our current implementation is correct.

If you want to change the aria announcements you can probably do that by leveraging the `actionsToolbarProps`
or `navigationActionsToolbarProps` props of the `DynamicPageTitle` component. There you can pass all allowed props of the `Toolbar` component.

**Comment by ksblklu** [comment]
Hi @Lukas742 
Thanks for the information! For the item count announcement of buttons inside `Toolbar`,  `onOverflowChange` method could solve this issue. But in [SAPUI5 Toolbar example](https://sapui5.hana.ondemand.com/?sap-ui-theme=sap_horizon#/entity/sap.m.Toolbar/sample/sap.m.sample.ToolbarResponsive), we saw there is an item count announcement implemented in `Toolbar` component overflow menu. Can we align our `Toolbar` component to `UI5 Toolbar`?
<img width="992" alt="image" src="https://user-images.githubusercontent.com/27538066/187362880-9415d3f7-d9c1-4e53-87d1-7e3804be6b49.png">

**Comment by Lukas742** [comment]
Hi @ksblklu 

The SAPUI5 `Toolbar` component has no overflow capabilities, so the item announcement is coming from the custom implemented `ActionSheet` (see [implementation details](https://sapui5.hana.ondemand.com/?sap-ui-theme=sap_horizon#/entity/sap.m.Toolbar/sample/sap.m.sample.ToolbarResponsive/code/ActionSheet.fragment.xml)) and not the `Toolbar` itself. We based our toolbar on the SAPUI5 [OverflowToolbar](https://sapui5.hana.ondemand.com/#/entity/sap.m.OverflowToolbar/sample/sap.m.sample.OverflowToolbarSimple)

**Comment by ksblklu** [comment]
Hi @Lukas742 
Thanks for the details, I'll use the `actionsToolbarProps` to customize the aria announcement of DynamicPageTitle actions toolbar.
